docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,182 @@
+        HeliographicStonyhurst [label="Heliographic Stonyhurst (HGS)\nHeliocentric Earth Equatorial (HEEQ)"]
+        HeliocentricEarthEcliptic [label="Heliocentric Earth Ecliptic (HEE)"]
+        GeocentricEarthEquatorial [label="Geocentric Earth Equatorial (GEI)"]
+        HeliographicCarrington [label="Heliographic Carrington (HGC)"]
+        Heliocentric [label="Heliocentric Cartesian (HCC)"]
+        HeliocentricInertial [label="Heliocentric Inertial (HCI)"]
+        Helioprojective [label="Helioprojective Cartesian (HPC)"]
+        GeocentricSolarEcliptic [label="Geocentric Solar Ecliptic (GSE)"]
+        ICRS -> HCRS
+        ICRS -> HeliocentricMeanEcliptic
+        HCRS -> ICRS
+        HCRS -> HeliographicStonyhurst
+        HeliocentricMeanEcliptic -> ICRS
+        HeliocentricMeanEcliptic -> HeliocentricEarthEcliptic
+        HeliocentricMeanEcliptic -> GeocentricEarthEquatorial
+        HeliographicStonyhurst -> HeliographicCarrington
+        HeliographicStonyhurst -> Heliocentric
+        HeliographicStonyhurst -> HCRS
+        HeliographicStonyhurst -> HeliocentricInertial
+        HeliographicCarrington -> HeliographicStonyhurst
+        Heliocentric -> Helioprojective
+        Heliocentric -> HeliographicStonyhurst
+        Helioprojective -> Heliocentric
+        HeliocentricEarthEcliptic -> HeliocentricMeanEcliptic
+        HeliocentricEarthEcliptic -> GeocentricSolarEcliptic
+        GeocentricSolarEcliptic -> HeliocentricEarthEcliptic
+        HeliocentricInertial -> HeliographicStonyhurst
+        GeocentricEarthEquatorial -> HeliocentricMeanEcliptic
+        subgraph cluster_astropy {
+                color=blue
+                fontcolor=blue
+                penwidth=2
+                label=<<b>Frames implemented in Astropy</b>>
+                ICRS
+                HCRS
+                HeliocentricMeanEcliptic
+                astropy [label="Other Astropy frames" shape=box3d style=filled]
+                geocentric [label="Earth-centered frames (including GEO)" shape=box3d style=filled]
+                astropy -> ICRS
+                geocentric -> ICRS
+                ICRS -> astropy
+                ICRS -> geocentric
+        }
+        subgraph cluster_sunpy {
+                color=crimson
+                fontcolor=crimson
+                penwidth=2
+                label=<<b>Frames implemented in SunPy</b>>
+                Helioprojective
+                Heliocentric
+                HeliographicStonyhurst
+                HeliographicCarrington
+                subgraph cluster_sunpy11 {
+                        color=chocolate
+                        fontcolor=chocolate
+                        label=<<b>Requires SunPy 1.1</b>>
+                        HeliocentricInertial
+                        HeliocentricEarthEcliptic
+                        GeocentricSolarEcliptic
+                        GeocentricEarthEquatorial
+                }
+        }
+        newrank=true
+   }
+
```

Comment:
Would be nice to have a link to the coordinate transformations bit of the docs here. (See <link> for the coordinate transformation documentation)

Reply:
It's already linked through the `sunpy.coordinates` link, just without any blatant indicator.

Reply:
I think that in this case that is sufficient, as `sunpy.coordinates` takes you to the most relevant docs.

Reply:
Welp, I ended up adding a sentence to advertise the new coordinate-system table too, so resolving this.